apps/dot-cdn/src/app/app.module.ts

```diff
@@ -17,7 +23,12 @@ import { HttpClientModule } from '@angular/common/http';
         DropdownModule,
         BrowserAnimationsModule,
         HttpClientModule,
-        RouterModule.forRoot([])
+        RouterModule.forRoot([]),
```

Comment:
This sucks, I had to do this because we have the `Router` class in `CoreWebService` to do some redirects but we shouldn't. Nothing to do here for now.